src/main/java/codeu/model/store/basic/ActivityStore.java

```diff
@@ -14,63 +15,62 @@
 
 public class ActivityStore {
 
-	/** Singleton instance of ActivityStore. */
-	private static ActivityStore instance;
-	
-	/**
-	 * Returns the singleton instance of ActivityStore that should be shared between all
-	 * servlet classes. Do not call this function from a test; use getTestInstance() instead.
-	 */
-	public static ActivityStore getInstance() {
-		if (instance == null) {
-			instance = new ActivityStore(PersistentStorageAgent.getInstance());		
-		}
-		return instance;
-	}
+  /** Singleton instance of ActivityStore. */
+  private static ActivityStore instance;
 
-	/**
-	 * Instance getter function used for testing. Supply a mock for PersistentStorageAgent.
-	 *
-	 * @param persistentStorageAgent a mock used for testing
-	 */
-	public static ActivityStore getTestInstance(PersistentStorageAgent persistentStorageAgent) {
-		return new ActivityStore(persistentStorageAgent);	
-	}
-	
-	/**
-	 * The PersistentStorageAgent responsible for loading Activities from and saving Activities
-	 * to Datastore.
-	 */
-	private PersistentStorageAgent persistentStorageAgent;
+  /**
+   * Returns the singleton instance of ActivityStore that should be shared between all
+   * servlet classes. Do not call this function from a test; use getTestInstance() instead.
+   */
+  public static ActivityStore getInstance() {
+    if (instance == null) {
+      instance = new ActivityStore(PersistentStorageAgent.getInstance());
+    }
+    return instance;
+  }
 
-	/** The in-memory list of Activities. */
-	private List<Activity> activities;
+  /**
+   * Instance getter function used for testing. Supply a mock for PersistentStorageAgent.
+   *
+   * @param persistentStorageAgent a mock used for testing
+   */
+  public static ActivityStore getTestInstance(PersistentStorageAgent persistentStorageAgent) {
+    return new ActivityStore(persistentStorageAgent);
+  }
 
-	/** This class is a singleton, so its constructor is private. Call getInstance() instead. */
-	private ActivityStore(PersistentStorageAgent persistentStorageAgent) {
-		this.persistentStorageAgent = persistentStorageAgent;
-		activities = new ArrayList<>();
-	}
+  /**
+   * The PersistentStorageAgent responsible for loading Activities from and saving Activities
+   * to Datastore.
+   */
+  private PersistentStorageAgent persistentStorageAgent;
 
-	/** Add a new activity to the current set of activities known to the application. */
-	public void addActivity(Activity activity) {
-		activities.add(activity);
-		//persistentStorageAgent.writeThrough(activity);
-	}
+  /** The in-memory list of Activities. */
+  private List<Activity> activities;
```

Comment:
Looks like this cannot be final since we have a setActivities() method, but I would init this here with the declaration instead of in the constructor

Reply:
I will change this. Will it be okay if that makes its initialization slightly different from the rest of the store classes, or should I change theirs to init in the member variable instead of the constructor instead?